packages/ember-routing/lib/location/history_location.js

```diff
@@ -150,7 +151,8 @@ export default EmberObject.extend({
    @param path {String}
   */
   pushState(path) {
-    let state = { path: path };
+    let id = `${this.stateCounter++}`;
```

Comment:
I feel like I had a good reason to force it to a string. Let me go back and see why I did that. It might not be necessary now.